addons/mail/static/src/chatter/web/chatter_patch.js

```diff
@@ -415,6 +420,9 @@ patch(Chatter.prototype, {
         const schedule = async (thread) => {
             await this.store.scheduleActivity(thread.model, [thread.id]);
             this.load(thread, ["activities", "messages"]);
+            if (this.props.hasParentReloadOnActivityChanged) {
+                await this.reloadParentView();
+            }
```

Comment:
Make sure load from the line above is not done twice due to the reload of the record and the manual call.
Otherwise the load should be moved in else clause.